### ISSUE TITLE
How to run `<env=var> <bin> <cmd>` ?

### ISSUE THREAD

**Issue by nielskrijger** [cosmtrek/air#186]
This might sound silly, but I've been struggling to set both an ENV variable and specify a command on the executable.

This:

```
bin = "./myapp serve"
```

...works by running the myapp bin with the `serve`-command.

And this:

```
full_bin = "ENV=dev ./myapp
```

...works by running the myapp bin with the env var.

Now I want to get both of best worlds, and initially I figured this would work:

```
full_bin = "ENV=dev ./myapp serve"
```

But alas, that simply runs the app without the `serve`-command.

Similarly, running with both bins configured:

```
bin = "./wanthave-api serve"
full_bin = "ENV=development ./wanthave-api"
```

runs the app without the `serve`-command. I assume it's just ignoring `bin` when `full_bin` has been defined.

I'm thinking I'm missing something stupid, but I'm scratching my head.

(I don't want to set ENV=dev using dotenv, I'm using cobra/viper for config mgt and didn't want to introduce another dep just to circumvent an issue with livereload)

**Comment by makiuchi-d** [comment]
It might be resolved by #180 (I'm not sure if it will be completed).

Or, try [arelo](https://github.com/makiuchi-d/arelo):
```
$ ENV=dev arelo -- ./myapp serve
```

**Comment by xiantang** [comment]
seems you can just define `full_bin = "ENV=dev ./myapp serve"`, and remove bin. 
it is work in my side 
![image](https://user-images.githubusercontent.com/34479567/142757388-4d16753b-10a5-4905-bcc5-f9c85b4dced4.png)

![image](https://user-images.githubusercontent.com/34479567/142757394-278274da-a14c-4c36-b0cb-27ee37edb515.png)

![image](https://user-images.githubusercontent.com/34479567/142757403-ae043d50-bebc-47fb-aba0-8ab64fed8be0.png)

if u have any questions. plz, reopen the issue. thanks
